content/10.methods.md

```diff
@@ -177,8 +182,7 @@ Full information about which countries comprised each region can be found in the
 
 ### Affiliation Analysis
 
-Along with the corresponding author names, we collected their affiliations recorded in each publication for this analysis.
-During the honoree curation process, if an honoree was listed with their affiliation at the time, we recorded this affiliation for analysis.
-For ISCB Fellows, we used the affiliation listed on the ISCB page.
-Because we could not find affiliations for the 1997 and 1998 RECOMB keynote speakers' listed for these years, they were left blank.
-If an author or speaker has more than one affiliation, each is inversely weighted by the number of affiliations that individual has.
+For each country, we computed the expected number of honorees by multiplying the proportion of authors whose affiliations are in that country with the total number of honorees.
+We also performed an enrichment analysis to examine the difference in country affiliation proportions between ISCB honorees and Pubmed corresponding authors.
+We calculated each country's enrichment by dividing the observed proportion of honorees by the expected proportion of honorees.
+The variance of the log~2~ enrichment is estimated using the delta method with a small continuity correction to avoid dividing by 0.
```

Comment:
```suggestion
The variance of the log~2~ enrichment was estimated using the delta method with a small continuity correction to avoid dividing by 0.
```